react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class NewsLetterForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: undefined };
```

Comment:
For input starting values it is better to start them as an empty string: `{ value: '' }`